modules/backend/routes.php

```diff
@@ -20,7 +20,7 @@
     Event::fire('backend.beforeRoute');
 
     /*
-     * Other pages
+     * Entry point
```

Comment:
Let's describe what is actually happening so it's clear
```suggestion
     * Route everything to `Backend\Classes\BackendController`, which itself acts as a router for the Backend.
```